p2p/server/msgserver.go

```diff
@@ -147,27 +148,37 @@ func (p *MessageServer) handleMessage(msg Message) {
 }
 
 func (p *MessageServer) handleRequestMessage(msg Message, data *service.DataMsgWrapper) {
-	if payload := p.msgRequestHandlers[MessageType(data.MsgType)](msg); payload != nil {
-		rmsg := &service.DataMsgWrapper{MsgType: data.MsgType, ReqID: data.ReqID, Payload: payload}
-		sendErr := p.network.SendWrappedMessage(msg.Sender(), p.name, rmsg)
-		if sendErr != nil {
-			p.Error("Error sending response message, err:", sendErr)
+	p.Debug("handleRequestMessage start")
+	if foo, okFoo := p.msgRequestHandlers[MessageType(data.MsgType)]; okFoo {
```

Comment:
in my opinion -
``` foo, okfoo := ...
if !okFoo {
retirm
}```
is cleaner, though you can't use the if := thing :)

Reply:
true